### PR TITLE
Api call for all except edit 

### PR DESCRIPTION
**Pull request by Developers-codz**
## Files to Check
`src/components/eachNote/EachNote.jsx`
`src/components/modal/EditorModal.jsx`
`src/pages/archive/Archive.jsx`
`src/pages/home/Home.jsx`
`src/pages/trash/Trash.jsx`
`src/Redux/Reducers/notesSlice.js`

### [Live](https://notesify-git-api-call-developers-codz.vercel.app/)

Thank you for reviewing my PR :)

### CODE REVIEW

**Comment by vishalg8454** [comment]
Found a bug which leads to crash. Fix it first.
Steps to reproduce
- login
- add notes
- add only title in note
- click on add note
- app crashes

**Comment by Developers-codz** [comment]
> Found a bug which leads to crash. Fix it first. Steps to reproduce
> 
> * login
> * add notes
> * add only title in note
> * click on add note
> * app crashes

Thanks for pointing out the bug, I fixed that. Can you please now review and approve :)

File: src/components/modal/EditorModal.jsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+              onChange={setTagsHandler}
+            />
+            <Checkboxlabel htmlFor="health">Health</Checkboxlabel>
+            <CheckBoxInput
+              type="checkbox"
+              checked={tags.some((tag) => tag === "Study")}
+              value="Study"
+              id="study"
+              onClick={setTagsHandler}
+            />
```
Comment (vishalg8454) [inline src/components/modal/EditorModal.jsx:161]: It's ok for now. But later on use a single checkbox component and map over some array.

File: src/pages/archive/Archive.jsx [hunk 2]
```diff
@@ -1,7 +1,49 @@
...
 
 export const Archive = () => {
+  const { archive } = useSelector((store) => store.notes);
+  const dispatch = useDispatch();
+  const [isLoading, setLoading] = useState(true);
+  useEffect(() => {
+    setTimeout(() => {
+      setLoading(false);
+    }, 800);
+  }, []);
```
Comment (vishalg8454) [inline src/pages/archive/Archive.jsx:21]: Seems like you are trying to show a fake loader?

**Review by vishalg8454 — APPROVED** [review]
Left a few comments, can merge.
Also the notes priority selector(while adding note) should default to `low` as the default value you have set is low only.

File: src/components/modal/EditorModal.jsx (hunk 1 above)
Comment (Developers-codz) [reply src/components/modal/EditorModal.jsx:161]: okay, will do that later

File: src/pages/archive/Archive.jsx (hunk 2 above)
Comment (Developers-codz) [reply src/pages/archive/Archive.jsx:21]: yep

Comment (vishalg8454) [reply src/pages/archive/Archive.jsx:21]: okay, but correct it later.

**Review by JatuFaique — APPROVED** [review]
The code looks clean and implementation is clear,
Looks good to merge.

**Pull request merged**